### PR TITLE
updated CONDUCT.md with links to our CoC

### PR DESCRIPTION
**Pull request by tracykteal**
Addresses https://github.com/swcarpentry/lesson-example/issues/115

### CODE REVIEW

**Comment by tracykteal** [comment]
@king @naupaka @k8hertweck @rgaiacs please review, and add LGTM or that it needs more discussion.

**Comment by k8hertweck** [comment]
This is excellent, thanks for the catch and the fix. LGTM (I had to google that and am proud I get to use it)

File: CONDUCT.md [hunk 1]
```diff
@@ -32,13 +32,14 @@ or other unprofessional conduct.
...
-may be reported by opening an issue or contacting one or more of the project maintainers.
+may be reported by following our [reporting guidelines](https://software-carpentry.org/CoC-reporting/).
 
-This Code of Conduct is adapted from
-the [Contributor Covenant][contrib-covenant] Version 1.0.0.
+
+- [Software and Data Carpentry Code of Conduct](https://software-carpentry.org/conduct/)
+- [Code of Conduct Reporting Guide](https://software-carpentry.org/CoC-reporting/)
 
 {% include links.md %}
```
Comment (rgaiacs) [inline CONDUCT.md]: Could you move all the links to the file links.md?

**Comment by tracykteal** [comment]
Links updated to use links.md

**Comment by naupaka** [comment]
LGTM

**Comment by rgaiacs** [comment]
@tracykteal I will copy your commits to https://github.com/swcarpentry/styles/ later this week. Thanks for the pull request.

**Comment by rgaiacs** [comment]
I'm closing this pull request since https://github.com/swcarpentry/styles/pull/147 was merged.

**Pull request closed without merging**